src/data/content/community/eventi.yaml

```diff
@@ -27,7 +27,7 @@ metadata:
 # ————————————————————————————————————————————
 seo:
   name: Eventi - Designers Italia #C REMIND "- Designers Italia" #AUTO
-  description: Tutti gli eventi del progetto Desingers Italia #C
+  description: Tutti gli eventi del progetto Designers Italia #C
```

Comment:
This is not about the video feature, but I guess it's so small of a change we can group it together anyway